### PR TITLE
Issue #PS-1662 feat: functionality changes for credential modal done

### PR DESCRIPTION
**Pull request by suvarnakale**

### CODE REVIEW

File: src/components/AddFacilitator.tsx [hunk 1]
```diff
@@ -50,7 +59,20 @@ const AddFacilitatorModal: React.FC<AddFacilitatorModalprops> = ({
   const [coreFields, setCoreFields] = React.useState<string[]>([]);
 
   const { t } = useTranslation();
-
+  const theme = useTheme<any>();
+  const style = {
+    position: 'absolute',
+    top: '50%',
+    left: '50%',
+    transform: 'translate(-50%, -50%)',
```
Comment (itsvick) [inline src/components/AddFacilitator.tsx:67]: @suvarnakale use common style

File: src/components/AddFacilitator.tsx [hunk 2]
```diff
@@ -262,54 +266,64 @@ const AddFacilitatorModal: React.FC<AddFacilitatorModalprops> = ({
...
+                    const credentialResponse = await sendCredentialService({
+                      isQueue,
+                      context,
+                      key,
+                      replacements,
+                      email: sendTo,
+                    });
+
+                    if (
+                      credentialResponse.result[0].data[0].status === 'success'
```
Comment (itsvick) [inline src/components/AddFacilitator.tsx:305]: @suvarnakale add ?.

Comment (suvarnakale) [reply src/components/AddFacilitator.tsx:305]: ok

File: src/components/AddFacilitator.tsx (hunk 1 above)
Comment (suvarnakale) [reply src/components/AddFacilitator.tsx:67]: ok

**Pull request merged**